src/controller/controller.js

```diff
@@ -0,0 +1,31 @@
+var model = require('../models/index');
```

Comment:
This, for now, is looking like utility methods to access the database, whereas controller code might be more like code you call for each specific use. So maybe these should be exported from something like `/src/storage/util.js`? Just a thought, i may not be right...